Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableColorTheme.cs

```diff
@@ -227,6 +239,24 @@ protected bool TryGetTextMeshProUGUIColor(ThemeStateProperty property, out Color
             return false;
         }
 
+        /// <summary>
+        /// Try to get color from Image
+        /// If no color is found, Image is not on the object
+        /// </summary>
+        protected bool TryGetUnityUiImageColor(ThemeStateProperty property, out Color color)
+        {
+            Color colour = Color.white;
```

Comment:
>colour [](start = 18, length = 6)

Rename or remove this variable.  It's odd for it to have the British spelling right next to a param with the American spelling.

I'd probably just remove it and use Color.white below where this is used.  Or you could rename it to defaultColor.